plenum/common/messages/node_messages.py

```diff
@@ -152,11 +151,14 @@ class PrePrepare(MessageBase):
         (f.TXN_ROOT.nm, MerkleRootField(nullable=True)),
         (f.SUB_SEQ_NO.nm, NonNegativeNumberField()),
         (f.FINAL.nm, BooleanField()),
+        (f.POOL_STATE_ROOT_HASH.nm, MerkleRootField(optional=True,
```

Comment:
It needs to be after BLS_MULTI_SIG

Reply:
Please add a test checking that a Primary sending POOL_STATE_ROOT_HASH is understood by nodes which are not aware of this field

Reply:
POOL_STATE_ROOT_HASH needs to be before BLS_MULTI_SIG because PrePrepare can be not contains multi signature.
But will add tests for parsing different variants of PrePrepare messages.

Reply:
- add test_process_pre_prepare_validation_old_schema  (for all variants of PrePrepare)
- change test_process_pre_prepare_validation (for all variants of PrePrepare)